backend/db/database.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE howls (
+    id SERIAL NOT NULL PRIMARY KEY,
+    caption VARCHAR(400),
+    createdAt timestamp with time zone
+);
```

Comment:
Let's add an updatedAt with TIMESTAMPTZ (same as `timestamp with time zone`)
I actually like that you named the text caption now that I think about it 😄
```suggestion
CREATE TABLE howls (
    id SERIAL NOT NULL PRIMARY KEY,
    caption VARCHAR(400),
    createdAt TIMESTAMPTZ DEFAULT current_timestamp
    updatedAt TIMESTAMPTZ DEFAULT current_timestamp
);
```
Not sure if all of these are necessary, but here is an example

Reply:
Added suggested changes and also modified the syntax a little! 😘